src/main/java/core/basesyntax/WorkRecord.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class WorkRecord {
+    private static final DateTimeFormatter DATE_FORMATTER = DateTimeFormatter
+            .ofPattern("dd.MM.yyyy");
+    private final LocalDate date;
+    private final String employeeName;
+    private final int hoursWorked;
+    private final int hourlyRate;
+
+    public WorkRecord(String record) {
```

Comment:
The constructor should not start with an empty line. Please remove the empty line above the constructor to adhere to Java best practices.